libs/vuepress/src/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { ExecutorContext } from '@nrwl/devkit';
+import * as path from 'path';
+import * as semver from 'semver';
+
+export function isVuepress2(context: ExecutorContext): boolean {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { devDependencies, dependencies } = require(path.join(
+    context.root,
+    'package.json'
+  )) as {
+    devDependencies?: { [key: string]: string };
+    dependencies?: { [key: string]: string };
+  };
+  let vuepressVersion = devDependencies?.vuepress || dependencies?.vuepress;
+
+  if (!vuepressVersion) {
+    throw new Error(
+      'Nx Plus failed to find vuepress listed in your package.json.'
+    );
+  }
+
+  if (vuepressVersion.startsWith('^') || vuepressVersion.startsWith('~')) {
```

Comment:
I noticed `semver` has a `coerce` method that you can use rather than doing this check.